scripts/style.sh

```diff
@@ -21,19 +21,61 @@
 # Commonly
 # ./scripts/style.sh master
 
-if [[ $(clang-format --version) != **"version 6"** ]]; then
+system=$(uname -s)
+
+if [[ $(clang-format --version) != *"version 6"* ]]; then
   echo "Please upgrade to clang-format version 6."
   echo "If it's installed via homebrew you can run: brew upgrade clang-format"
   exit 1
 fi
 
-if [[ $# -gt 0 && "$1" = "test-only" ]]; then
+if [[ "$system" == "Darwin" ]]; then
+  version=$(swiftformat --version)
+  version="${version/*version /}"
+  # Allow an older swiftformat because travis isn't running high sierra yet
```

Comment:
Nit: please capitalize High Sierra and Sierra

Reply:
Done.